docs/arch/README.md

```diff
@@ -0,0 +1,66 @@
+## MessageHub Trigger Provider Architecture
+
+### Create Trigger Feed
+![MessageHub Trigger Create](images/Arch-Provider-MHV1-Create.png)
+
+**Scenario:** User wants to create a trigger `trigger1` for MessageHub service instance `instance1`, using Credentials `Credential-1` and use rule `rule1` to invoke action `action1` with messages from topic `topic1`.
+
+1. Developer creates a MessageHub service `instance1`
+2. Developer creates topic `topic1` in MessageHub service `instance1`
+3. Developer creates Credential key `Credential-1` for MessageHub `instance1`
+4. Developer creates trigger `trigger1` on OpenWhisk, the trigger stores the annotation `feed` with the feedAction name from system package or binded package.(`/whisk.system/messagingWeb/messageHubFeed`).
+5. Developer invokes action feedAction to create trigger feed passing input parameters (lifeCycle:`CREATE`, trigger1, Credentials1, Options:`topic1`)
+6. The feedAction invokes feedWebAction forwarding input parameter.
+7. The feedWebAction inserts trigger feed doc into DB for worker group 0 (feedWebActions protects DB credentials)
+8. DB insertion notifies workers group 0 via Cloudant/CoucDB changes API, workers listen on DB view with a filter for their group `worker0` and gets the DB doc.
+9. Kafka Consumer is created on each worker in a consumer group and starts polling for messages on `topic1` from `instance1` using `Credentials-1`.
+10. Developer creates rule1 indicating that when trigger1 fires invoker action1
```

Comment:
Add backtick symbol: Developer creates `rule1` indicating that when `trigger1` fires invoker `action1`